eng/.docsettings.yml

```diff
@@ -46,17 +47,17 @@ required_readme_sections:
   - ^Contributing$
 
 known_presence_issues:
-  - ["CHANGELOG.md", "#1583"]
-  - ["sdk/applicationinsights/applicationinsights-query/CHANGELOG.md", "#1583"]
-  - ["sdk/batch/batch/CHANGELOG.md", "#1583"]
-  - ["sdk/eventgrid/eventgrid/CHANGELOG.md", "#1583"]
-  - ["sdk/graphrbac/graph/CHANGELOG.md", "#1583"]
-  - ["sdk/operationalinsights/loganalytics/CHANGELOG.md", "#1583"]
-  - ["sdk/servicefabric/servicefabric/CHANGELOG.md", "#1583"]
-  - ["sdk/storage/storage-datalake/CHANGELOG.md", "#1583"]
-  - ["sdk/template/template/CHANGELOG.md", "#1583"]
-  - ["sdk/test-utils/recorder/CHANGELOG.md", "#1583"]
-  - ["sdk/test-utils/perfstress/CHANGELOG.md", "#1583"]
+  - ["CHANGELOG.md", "6421"]
```

Comment:
Why the switch to 6421? That appears to be a merged PR.

Reply:
Yes it is a merged PR by Dozie - where he added these changelog items - so I updated it - so that we can track why these got added in here. cc : @chidozieononiwu

Reply:
@chidozieononiwu Do you remember why you had added these changelog items as part of #6421 PR - do you still need them?

Reply:
That change was made as prep or updated that was intended to make `docwarden` also operate on `CHANGELOG.md` files, However it was discarded since the concept of `eng/common` was introduced and changelog updated eventually went there.
That change makes no difference whatso ever now that changelog updates did not make it into docwarden,  you can remove all the CHANGELOG.md links and ther will be no problem since docwarden only looks at README.md files.

Reply:
Actually the right issue should be `https://github.com/Azure/azure-sdk-for-js/issues/1583` which basically just says that if you add to this list you should eventually fix up the readme

Reply:
@chidozieononiwu Yes but #1583 is for readme and these lines have changelog items - so I removed the changelog items since these are not required here anymore - cleaning it up!